tests/cucumber/features/step_definitions/shared_steps.coffee

```diff
@@ -0,0 +1,55 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
```

Comment:
@aslagle I just spent a few hours trying to get these two lines to automatically run before all tests, since I think we should always be clearing the database and starting from the homepage (I was getting weird errors, since some tests were starting where the previous test had ended and trying to refresh the page after clearing the database).  I couldn't get this.Before to work at all.